sympy/vector/operators.py

```diff
@@ -240,6 +240,8 @@ def divergence(vect, coord_sys=None, doit=True):
     if len(coord_sys) == 0:
         return S.Zero
     elif len(coord_sys) == 1:
+        if isinstance(vect, (Cross, Curl, Gradient)):
+            return Divergence(vect)
```

Comment:
Do we need this condition? I'm not so sure, support for the derivatives could be added for `Cross` and `Curl`.